browser-test/src/applicant/applicant_application_index.test.ts

```diff
@@ -317,9 +318,13 @@ test.describe(
       )
     })
 
-    test('categorizes programs for draft and applied applications as guest user', async () => {
+    test('categorizes programs for draft and applied applications', async () => {
       const {applicantQuestions, page} = ctx
 
+      await disableFeatureFlag(page, 'north_star_applicant_ui')
```

Comment:
**nitpick:** Maybe add a TODO to remove this disabling once login is handled in the north star work / and same below for logout

Reply:
Done